### PR TITLE
fix(markdown-legacy): normalize some characters for kramed's annotate function

### PR DESCRIPTION
**Pull request by solufa**
fix: https://github.com/honkit/honkit/pull/85#discussion_r449770208

### CODE REVIEW

File: packages/@honkit/markdown-legacy/lib/page.js [hunk 1]
```diff
@@ -33,9 +33,7 @@ function combine(nodes) {
     @return {String}
 */
 function preparePage(src) {
-    // annotate.blocks does not normalize src, so windows fail the reason
-    const normalizedSource = src.replace(/\r\n|\r/g, "\n");
-    var lexed = annotate.blocks(normalizedSource);
+    var lexed = annotate.blocks(src);
```
Comment (azu) [inline packages/@honkit/markdown-legacy/lib/page.js:36]: We can not require that user should not use `CRLF`.
So, Its needed in parser.

I think that this is a bug of kramed.
kramed does same normalize in lexer.
https://github.com/GitbookIO/kramed/blob/9d96b7ac9e063b94d51423d8cd450f4a7c7eb1f3/lib/lex/block.js#L50-L58
But, kramed's `annotate` function does not it.

**Comment by azu** [comment]
Thanks PR. I've commented

File: packages/@honkit/markdown-legacy/lib/page.js (hunk 1 above)
Comment (azu) [reply packages/@honkit/markdown-legacy/lib/page.js:36]: 📝 I've noticed that we need to add [other normalize](https://github.com/GitbookIO/kramed/blob/9d96b7ac9e063b94d51423d8cd450f4a7c7eb1f3/lib/lex/block.js#L53-L55) into this line OR fork kramed.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ jobs:
         node-version: [10, 12, 14]
         os: [macos-latest, windows-latest, ubuntu-18.04]
     steps:
+      - name: Disable autocrlf
+        run: git config --global core.autocrlf false
+        if: matrix.os == 'windows-latest'
```
Comment (azu) [inline .github/workflows/test.yml:16]: Can you remove this?
We want to use default config.

**Comment by solufa** [comment]
Thanks for the comments.
I didn't pay enough attention to users.

Sorry for the many commits.
I can rebase it if necessary.

**Comment by azu** [comment]
@solufa No problem! Thanks for PR

**Pull request merged**